code/frameworks/react-vite/package.json

```
@@ -47,7 +47,7 @@
     "prep": "../../../scripts/prepare/bundle.ts"
   },
   "dependencies": {
-    "@joshwooding/vite-plugin-react-docgen-typescript": "0.2.1",
+    "@joshwooding/vite-plugin-react-docgen-typescript": "0.3.0",
```

```suggestion
    "@joshwooding/vite-plugin-react-docgen-typescript": "0.3.1",
```
This version also moves vite imports to dynamic imports